src/pytest_molecule/__init__.py

```diff
@@ -173,6 +173,13 @@ def __init__(self, name, parent):
             self.molecule_driver = data.get("driver", {}).get("name", "no_driver")
             self.add_marker(self.molecule_driver)
 
+            # check for tags and add them as markers
+            for tag in data.get("tags", []):
+                try:
+                    self.add_marker(tag)
```

Comment:
Some markers have arguments, like conditionals and behavior modifiers. It looks like adding them unconditionally doesn't account for that.

Reply:
That is outside the scope of this change. I have no plans to add support for conditionals, not myself. If someone does I will review but I do not find it useful enough. Ability to temporarily mark a test as allowed to fail is enough for me.